esp-hal/src/debugger.rs

```diff
@@ -0,0 +1,39 @@
+//! Debugger utilities
+
+/// Checks if a debugger is connected.
+pub fn debugger_connected() -> bool {
+    #[cfg(xtensa)]
+    {
```

Comment:
I think we should use method we have in `xtensa-lx` instead of duplicating the code here.

Reply:
good call